Foundation/NSAffineTransform.swift

```diff
@@ -63,42 +76,30 @@ public class NSAffineTransform : NSObject, NSCopying, NSSecureCoding {
     
     // Transforming points and sizes
     public func transformPoint(aPoint: NSPoint) -> NSPoint {
-        let matrix = transformStruct.matrix3x3
-        let vector = Vector3(aPoint.x, aPoint.y, CGFloat(1.0))
-        let resultVector = multiplyMatrix3x3(matrix, byVector3: vector)
-        return NSMakePoint(resultVector.m1, resultVector.m2)
+        /**
+         [ x' ]     [ x ]   [ m11  m12  tX ] [ x ]   [ m11*x + m12*y + tX ]
+         [ y' ] = T [ y ] = [ m21  m22  tY ] [ y ] = [ m21*x + m22*y + tY ]
+         [  1 ]     [ 1 ]   [  0    0    1 ] [ 1 ]   [           1        ]
+         */
+        let x = transformStruct.m11*aPoint.x + transformStruct.m12*aPoint.y + transformStruct.tX
+        let y = transformStruct.m21*aPoint.x + transformStruct.m22*aPoint.y + transformStruct.tY
```

Comment:
i think readability can be improved here with some spacing and `()` :smile: 

```swift
let x = (transformStruct.m11 * aPoint.x) + (transformStruct.m12 * aPoint.y) + transformStruct.tX
```

(same below)